using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details.md

```diff
@@ -0,0 +1,83 @@
+| Content type       | contenttype  | string | *(Optional)* Used to set the Content-Type header for POST requests. If the header is not supplied, but a request body is found, the default is `application/x-www-form-urlencoded`. |
+| Connection timeout | timeout      | number | *(Optional)* The timeout value for the network connection in seconds. The default value is 2 seconds. |
+
+## Sync PII consequence
+
+This rule is currently handled by the [Signal](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/signals) extension.
+
+| Friendly name      | Key          | Type   | Description                                                  |
+| ------------------ | ------------ | ------ | ------------------------------------------------------------ |
+| Description URL    | templateurl  | string | *(Required)* Destination URL to which the PII signal will be sent. Must be HTTPS, and non-HTTPS URLs will be ignored. |
+| Request body       | templatebody | string | *(Optional)* A base64-encoded blob that contains the POST body to be sent. If this value exists, the PII signal will be sent as a POST instead of as a GET request. |
+| Content type       | contenttype  | string | *(Optional)* Used to set the Content-Type header for POST requests. If the header is not supplied, but a request body is found, the default is `application/x-www-form-urlencoded`. |
+| Connection timeout | timeout      | number | *(Optional)* The timeout value for the network connection in seconds. The default value is 2 seconds. |
+
+## Open URL consequence
+
+This rule is currently handled by the [Signal](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/signals) extension.
+
+| Friendly name | Key  | Type   | Description                                                  |
+| ------------- | ---- | ------ | ------------------------------------------------------------ |
+| URL           | url  | string | *(Required)* URL that will be opened by the target platform. This might be a URL for a website or a deep link to an app. |
+
+## Profile consequence
+
+This rule is currently handled by the [Profile](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/profile) extension.
+
+| Friendly name             | Key       | Type             | Description                                                  |
+| ------------------------- | --------- | ---------------- | ------------------------------------------------------------ |
+| Operation to perform      | operation | string           | *(Required)* Determines the type of operation to be performed on the profile. The following operations are supported: <br/>- *write,* which saves `value` in the given `key` in the shared state and the local profile.<br/>If an associated value for the key already exists, the existing value will be overwritten. <br/> - *delete,* which removes `key` from the profile's shared state and the local profile. |
```

Comment:
In the description section, is it possible to line the sentence that starts with "If an associated value..." under the start of the text of the "write" bullet? Like this:
"- write, which saves..."
    If an associated value for the key..." 
(If takes too much time to fix it, leave it as is :-)!)

Reply:
I added the break \<br/>, so currently it looks like this:

<img width="500" alt="Screen Shot 2019-11-26 at 8 55 54 PM" src="https://user-images.githubusercontent.com/33132425/69694898-44de2b80-108f-11ea-859b-81487b0543fa.png">

Is this what you suggested?

Reply:
very close @emdobrin..I was hoping that it might be .possible to move the "If an associated value..." sentence over to under the start of "write"; if not, this is fine. Thanks!

Reply:
I think it worked, hopefully the change will be preserved in gitbook.

![Screen Shot 2019-11-27 at 9 57 50 AM](https://user-images.githubusercontent.com/33132425/69748106-7c36f180-10fc-11ea-8495-e4f572d95602.png)